wolfssl/wolfcrypt/types.h

```diff
@@ -202,7 +202,11 @@ decouple library dependencies with standard string, memory and so on.
             #define WC_INLINE
         #endif
     #else
-        #define WC_INLINE
+        #ifdef __GNUC__
+            #define WC_INLINE __attribute__((unused))
```

Comment:
I this to allow inlined functions to be unused? How many cases of this did you see? Seems like an okay workaround to the `-Wall`.

Reply:
right, just to avoid warnings on unused static inlines.